pages/public_cloud/ai_machine_learning/notebook_guide_concepts_notebooks/guide.en-gb.md

```diff
@@ -0,0 +1,64 @@
+---
+title: AI Notebooks - Notebook concept
+excerpt: Learn the concept behind AI Notebooks
+updated: 2023-11-20
+---
+
+## Definition
+
+An **AI Notebook** is the workload unit submitted to the cluster. **AI notebooks** are managed Jupyter or VSCode notebooks, with pre-packaged Machine Learning frameworks linked to computing resources (CPU or GPU) and storage space. They are running within OVHcloud infrastructure.
+
+Each AI Notebook is linked to a **Public Cloud** project and specifies the following:
+- a type and quantity of computing resources
+- a code editor
+- a pre-configured Machine Learning framework
+
+## Considerations
+
+> [!warning]
+>
+> An **AI Notebook** runs continuously until manually interrupted by the user, unless it exceeds **7 days of running**. It will then be automatically stopped. You can choose to automatically restart it using the `auto-restart` option (set this parameter to `True`).The notebook will then restart as is.
```

Comment:
```suggestion
> An **AI Notebook** runs continuously until manually interrupted by the user, unless it exceeds **7 days of running**. It will then be automatically stopped. You can choose to automatically restart it using the `auto-restart` option (set this parameter to `True`). The notebook will then restart as is.
```